src/modifiers/changeCurrentInlineStyle.js

```diff
@@ -8,19 +9,28 @@ const changeCurrentInlineStyle = (editorState, matchArr, style) => {
   const { index } = matchArr;
   const blockMap = currentContent.getBlockMap();
   const block = blockMap.get(key);
-  const currentInlineStyle = block.getInlineStyleAt(index).merge();
-  const newStyle = currentInlineStyle.merge([style]);
+  const currentInlineStyle = block.getInlineStyleAt(index);
+  // do not modify the text if it is inside code style
+  const hasCodeStyle = currentInlineStyle.find(style => style === "CODE");
+  if (hasCodeStyle) {
+    return editorState;
```

Comment:
I see where you were going with this, but it might be less edge casey to filter all the text that's styled as inline code so it doesn't apply to the other inline style calculations maybe? Thoughts?

https://github.com/withspectrum/draft-js-markdown-plugin/issues/96#issuecomment-400343639

Reply:
I.e. instead of just ignoring this text that should be italic (which I think would happen rn?)

```
this is `some _code` and some more _text_
```

we could just remove the inline code bit and everything before it and pretend that the text is

```
 and some more _text_
```

Does that make sense or am I way overcomplicating things?